client/app/components/button-widget/component.js

```diff
@@ -4,26 +4,26 @@ import Ember from 'ember';
 export default Ember.Component.extend({
 
     didReceiveAttrs() {
-        this.set('widgetClasses', this.attrs.widget.value.css_classes)
+        this.set('widgetClasses', this.attrs.widget.value.cssClasses)
     },
 
     buttonString: 'Save',
 
     widgetClasses: ['section-submit-button'],
     widgetClassString: Ember.computed('widgetClasses', function() {
-        let classArr = this.get('widgetClasses')
-        if (classArr === undefined ||
-            classArr.constructor !== Array
+        const CLASSES = this.get('widgetClasses')
```

Comment:
It looks like you are using uppercase for constants, which does stem from a naming convention but it's not in Airbnb style because the new ES6 const is a different than a variable that shouldn't be changed across the script. Check out this discussion: https://github.com/airbnb/javascript/pull/255

I would recommend returning these uppercase to lowercase when using const except if they are truly constants like keys, numbers the entire script should use, in otherwise beyond scope of individual functions.